ironfish-rust/src/mining/mine.rs

```diff
@@ -64,11 +64,11 @@ mod test {
         let start = 42;
         let step_size = 1;
 
-        // Hardcoded target value derived from a randomness of 1, which is lower than 42
+        // Hardcoded target value derived from a randomness of 43, which is lower than 42
```

Comment:
Does this comment need updating to say 43 is higher than 42? I'm not 100% sure what's going on here

Reply:
Yeah.. technically, 43 is correct and 1 was incorrect. It is awkwardly worded, for sure. tldr: I run the test with a target of 0 and just print the target for randomness 43, then update it here. Honestly don't love the test either but wanted to test the batching

Reply:
Ah gotcha, I understand it now, thanks!